scripts/tests/cases/store_benchmark.py

```diff
@@ -0,0 +1,716 @@
+
+            except Exception as e:
+                print(f"Note: Could not get detailed chunk counts: {e}")
+                metrics["total_elements"] = "N/A"
+                metrics["text_chunks"] = "N/A"
+                metrics["table_chunks"] = "N/A"
+                metrics["chart_chunks"] = "N/A"
+
+            # Check Milvus statistics
+            try:
+                milvus_uri = f"http://{self.hostname}:19530"
+                milvus_chunks(milvus_uri, collection_name)
+            except Exception as e:
+                print(f"Warning: Could not get Milvus stats: {e}")
+
+            # Test retrieval
+            retrieval_start = time.time()
+            try:
+                test_queries = [
+                    "What is the main topic discussed?",
+                    "What financial information is provided?",
+                    "What are the key findings?",
+                ]
+
+                query_results = nvingest_retrieval(
+                    test_queries,
+                    collection_name,
+                    hybrid=True,
+                    embedding_endpoint=f"http://{self.hostname}:8012/v1",
+                    embedding_model_name=self.model_name,
+                    model_name=self.model_name,
+                    top_k=5,
+                    gpu_search=False,
+                )
+
+                metrics["retrieval_time"] = time.time() - retrieval_start
+                metrics["retrieval_success"] = True
+                metrics["query_count"] = len(test_queries)
+
+                # Check for MinIO URLs in results if store was enabled
+                if store_enabled:
+                    urls_found = self.check_minio_urls(query_results)
+                    metrics["minio_urls_found"] = urls_found
+                    print(f"MinIO URLs found in results: {urls_found}")
+
+                print(f"Retrieval time: {metrics['retrieval_time']:.2f}s")
+
+            except Exception as e:
+                print(f"Retrieval failed: {e}")
+                metrics["retrieval_time"] = None
+                metrics["retrieval_success"] = False
+                metrics["retrieval_error"] = str(e)
+
+        except Exception as e:
+            print(f"Ingestion failed: {e}")
+            metrics["ingestion_success"] = False
+            metrics["ingestion_error"] = str(e)
+            return metrics
+
+        # Store validation if enabled
+        if store_enabled:
+            try:
+                minio_stats = self.validate_minio_storage()
+                metrics.update(minio_stats)
+            except Exception as e:
+                print(f"MinIO validation failed: {e}")
+                metrics["minio_validation_error"] = str(e)
+
+        # Clean up temporary files after analysis
+        try:
+            import shutil
+
+            temp_dir = f"/raid/jioffe/tmp/store_task/{self.run_id}_{collection_name}"
+            image_dir = f"/raid/jioffe/tmp/store_task/{self.run_id}_{collection_name}_images"
```

Comment:
Related to the previous comment, if `temp_dir` is changed because user is able to set it different, it should change here too.